src/services/EtherspotSessionStorage.ts

```diff
@@ -0,0 +1,32 @@
+import { SessionStorage } from '@etherspot/prime-sdk';
+
+class LocalSessionStorage extends SessionStorage {
+  constructor () {
+    super();
+  }
+
+  setSession = async (walletAddress: string, session: Object) => {
+    if (walletAddress) {
+      window.localStorage.setItem(`session-prime-${walletAddress}`, JSON.stringify(session))
+    }
+  }
+
+  getSession = (walletAddress: string) => {
+    let result = null;
+
+    try {
+      const raw = window.localStorage.getItem(`session-prime-${walletAddress}`);
+      result = raw ? JSON.parse(raw) : null
+    } catch (err) {
+      //
```

Comment:
We should add a warning or something here should the device or security not allow local storage, feels like a showstopper?

Reply:
session storage removed completely per SDK team feedback